irr/include/SkinnedMesh.h

```diff
@@ -335,5 +304,38 @@ class SkinnedMesh : public IAnimatedMesh
 	bool HardwareSkinning;
 };
 
+// Interface for mesh loaders
+class SkinnedMeshBuilder : public SkinnedMesh {
+public:
+	SkinnedMeshBuilder() : SkinnedMesh() {}
+
+	//! loaders should call this after populating the mesh
```

Comment:
add `// returns *this, so do not try to drop the mesh builder instance`
this was confusing to me